media/css/mozorg/home/home-promo.less

```diff
@@ -1004,6 +1004,24 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
+// ViewSource Conf
+.promo-large-landscape.viewsource {
+    background-color: #00b085;
+
+    h2 {
+        padding-top: 135px;
```

Comment:
We should reduce this padding a little bit to move the text closer to the logo, which also means the circled-arrow thingie won't be quite as close to the bottom edge. Maybe 120px?